### PR TITLE
Add support for template definition inside of view configuration (ui-router)

### PR DESCRIPTION
**Pull request by avishnyak**
Addresses issue #28  …
Enhances handling of default exports
Added handling for all top-level ES module exports (there are others but they are less common for AngularJS)
Fixed a bug when parsing a component with ```bindings: true``` to isolate scope instead of actually providing a binding definition object
Added relevant tests

### CODE REVIEW

File: src/utils/route/routeParser.ts [hunk 1]
```diff
@@ -54,9 +54,31 @@ export class RouteParser {
...
+		if (viewsNode !== undefined) {
+			const viewConfigNodes = new ConfigParser(viewsNode).entries();
+
+			for (const viewNode of viewConfigNodes) {
+				const viewRoute = new Route();
+				viewRoute.name = `${route.name}[${viewNode[0]}]`;
+				viewRoute.pos = this.tsParser.sourceFile.getLineAndCharacterOfPosition(viewNode[1].pos);
+				viewRoute.path = this.tsParser.path;
+
+				if (ts.isStringLiteral(viewNode[1])) {
```
Comment (ipatalas) [inline src/utils/route/routeParser.ts]: This is nice. I didn't know such type guards exist in TS library. This seems better than my implementation of `isTsKind`. Keep using it. You can start replacing `isTsKind` with this wherever you're changing anything. I like it very much, it's more concise and self documenting at once.

**Comment by avishnyak** [comment]
@ipatalas Don't know if you are waiting for anything else from me or just busy with other stuff.

Unless you want me to make changes, I think this is ready to go whenever you have time.

**Comment by ipatalas** [comment]
I'm sorry. I'm just very busy recently. It can take few more weeks at most. 
At first sight it looks ok so this will probably get merged but I don't have time to make the release anyway at the moment. 
Thanks for contributing. This means a lot, it's just slightly bad timing now.

**Comment by avishnyak** [comment]
No problem! Good luck with your work.

**Pull request merged**